dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcIntegrationTest.java

```diff
@@ -348,11 +311,88 @@ void givenSomeSentFdcContributions_whenProcessDailyFilesRuns_thenTheyAreNotQueri
+		runProcessDailyFilesAndCheckResults(updatedIds, true, false, false, false, FdcContributionsStatus.WAITING_ITEMS);
+	}
+
+	/**
+	 * <h4>Scenario:</h4>
+	 * <p>A negative FDC Contributions test which checks that FDC Contribution records do not get picked up for processing by the Fast Track pickup logic,
+	 * 	if there are no Rep Order Crown Court Outcomes linked to the Rep Orders on which the FDC Contribution records are based.</p>
+	 * <h4>Given:</h4>
+	 * <p>* 3 fdc_contributions record IDs that would normally get picked up by the Fast Track pickup logic,
+	 * 		but their Rep Orders are missing the corresponding Crown Court Outcomes</p>
+	 * <h4>When</h4>
+	 * <p>* The {@link FdcService#processDailyFiles()} method is called</p>
+	 * <h4>Then:</h4>
+	 * <p>1. The call to the callGlobalUpdate is successful i.e. MAAT API returned a successful response
+	 * <p>2. The IDs of the 3 updated records are NOT returned.</p>
+	 * <p>3. The updated IDs are NOT included in the list of IDs returned by the call to retrieve 'REQUESTED' FDC Contributions.</p>
+	 * <p>4. The updated IDs are NOT included in the set of payloads sent to the DRC.</p>
+	 * <p>5. After the `processDailyFiles` method call returns, the fdc_contribution entities corresponding to each
+	 *       of the updated IDs is checked:<br>
+	 *       - Each remains at status WAITING_ITEMS<br>
+	 *       - Each has an unpopulated contribution_file ID.</p>
+	 *
+	 * @see <a href="https://dsdmoj.atlassian.net/browse/DCES-408">DCES-408</a> for test specification.
+	 */
+	//TODO: Fix test with implementation of /assessment/ endpoint access.
+	@Disabled("Pending creation of /assessment/ handler")
+	@Test
+	void givenFastTrackFdcContributionsWithMissingCCO_whenProcessDailyFilesRuns_thenTheirStatusIsNotUpdated() {
+		final var updatedIds = spyFactory.createFastTrackTestData(FdcAccelerationType.POSITIVE, FdcTestType.NEGATIVE_CCO, 3);
+		runProcessDailyFilesAndCheckResults(updatedIds, true, false, false, false, FdcContributionsStatus.WAITING_ITEMS);
+	}
+
+	/**
+	 * Private method to run the method under test and check the outcome with the provided criteria
+	 * @param updatedIds The IDs for the test data created before running the test (the Given bit)
+	 * @param stubDrcSuccessFlag Do we want to stubbed DRC client to succeed or fail for this test
+	 * @param updatedIdsShouldBeRequested Are the updated IDs expected to be requested for this test
+	 * @param updatedIdsShouldBeSent Are the updated IDs expected to be sent for this test
+	 * @param checkContributionFile Is the contribution file to be checked for this test
+	 * @param fdcContributionsStatusExpected The status expected at the end of this test
+	 */
+	private void runProcessDailyFilesAndCheckResults(
+			Set<Integer> updatedIds,
+			boolean stubDrcSuccessFlag,
+			boolean updatedIdsShouldBeRequested,
+			boolean updatedIdsShouldBeSent,
+			boolean checkContributionFile,
```

Comment:
Four `boolean` arguments in a row is going to be confusing to read in calls to this method (although I guess if you always use boolean literals, then IntelliJ will show the method parameter names inline).

Is there a better name for `checkContributionFile` since it seems to actually mean _don't check that it's null_?